starter-code/functions-and-arrays.js

```diff
@@ -122,32 +165,38 @@ var words = [
   "disobedience",
   "matter"
 ];
-
+function howManyTimes(array, search){
+  var count = 0;
+  words.forEach(function(name){
+    if (name === search){
+      count += 1
+    }
+  })
+  return count;
+}
 var howManyMatter = howManyTimes(words, "matter");
 console.log(howManyMatter);
+// 4
 
 var howManyDog = howManyTimes(words, "dog");
 console.log(howManyDog);
-
-// Bonus Quest
-function greatestProduct (matrix) {
-
-}
+// 0
+///BONUS
 
 var matrix = [
```

Comment:
¿Estáis seguros de que pastilla tomaréis cuando Morpheo llame a vuestras puertas?. Bien hecho!!!